pinc/ProjectSearchForm.inc

```diff
@@ -174,10 +202,17 @@ class ProjectSearchForm
         $this->define_form_widgets();
     }
 
+    function all_special_day_options()
+    {
+        return array(
+            '' => _('Any day'),
+            'Birthday' => _('Birthday'),
+            'Otherday' => _('Otherday'),
+        ) + $this->special_day_options();
+    }
+
     static public function special_day_options()
     {
-        $special_day_options = array();
```

Comment:
You need to keep this so that `$special_day_options` is pre-defined as an array before you append to it in the while loop further down.

Reply:
Yes. Changed this now.